src/visualization/types/Graph/view.tsx

```diff
@@ -201,7 +202,12 @@ const XYPlot: FC<Props> = ({
       colorMappingForGiraffe,
       colorMappingForIDPE,
       needsToSaveToIDPE,
-    } = getColorMappingObjects(fillColumnMap, properties)
+    } = getColorMappingObjects(fillColumnMap, {
+      ...properties,
+      colors: properties.colors
+        ? properties.colors
+        : (DEFAULT_LINE_COLORS as DashboardColor[]),
+    })
```

Comment:
I think we should consider memoizing this. I get why we're doing the copy operation with the `...` operator, but this is in the render loop and re-copying the properties object is a pretty inefficient operation

Reply:
True... let me rethink.